tests/test_data_utils.py

```diff
@@ -0,0 +1,43 @@
+import datetime as dt
+import os
+
+import numpy as np
+
+from sentinelhub import read_data
+from sentinelhub.data_utils import get_failed_statistical_requests, statistical_to_dataframe
+
+column_types = {
+    "ndvi_B0_min": float,
+    "ndvi_B0_max": float,
+    "ndvi_B0_mean": float,
+    "ndvi_B0_stDev": float,
+    "ndvi_B0_sampleCount": np.int64,
+    "ndvi_B0_noDataCount": np.int64,
+    "ndvi_B0_percentiles_50.0": float,
+    "ndvi_B0_bins": list,
+    "ndvi_B0_counts": list,
+    "interval_from": dt.date,
+    "interval_to": dt.date,
+    "identifier": str,
+}
```

Comment:
rewrite so that it is a sequence of pairs like `(float, ["ndvi_B0_min", "ndvi_B0_max", "ndvi_B0_mean", ...])`